test/ldap/README.md

```diff
@@ -0,0 +1,23 @@
+## Usage
+
+Building:
+
+```
+docker compose build
+docker compose up
+```
+
+The system should be available on `127.0.0.1:389` and `127.0.0.1:636` - with the creds `Administrator:admin123!` with `DEV-AD` the domain.
+
+Example of running a wih a Metasploit module:
+
+```msf
+msf6 auxiliary(scanner/ldap/ldap_login) > run rhost=127.0.0.1 username=DEV-AD\\Administrator password=admin123! CreateSession=true
```

Comment:
For this particular interactive ldap example snippet you might need to enable the ldap session types if you've not enabled it, i.e. with `features` and `features set ldap_session_type true`